### DISCUSSION TITLE
update pdfjs-dist version ?

### DISCUSSION THREAD

**Discussion by GreatAuk** [hrynko/vue-pdf-embed#150] [Q&A] [answered]
current v2.9.359, latest version is v3.8.162.

**Comment by markus-ebner** [comment]
@hrynko do you have any plans to switch to the major version 3 of pdfjs? 
Are there breaking changes to expect?

**Reply by hrynko** [reply]
I do, but there were certain issues with the forms after the previous updates (see #8), so I put it off for a while

**Comment by leeobrum** [comment]
@hrynko 
Can you answer if there will be this pdfjs update?

**Reply by hrynko** [reply]
There has been such an update, `pdfjs-dist@3.11.174` has been used since `v2`

**Comment by hrynko** [comment, marked as answer]
Upd: PDF.js has been updated to `3.11.174` in `v2.0.0` and to `4.3.136` in `v2.0.4`
